ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/system.cfg

```diff
@@ -11,10 +11,15 @@ interface Management1
 system control-plane
    tcp mss ceiling ipv4 1344 ipv6 1366
    ip access-group acl4_1 in
-   ip access-group acl4_3 vrf default in
    ip access-group acl4_2 vrf red in
    ip access-group acl4_2 vrf red_1 in
+   ip access-group acl4_3 vrf default in
+   ip access-group ingress default acl4_4
+   ip access-group acl4_5 vrf red_3 in
+   ip access-group ingress vrf red_4 in
```

Comment:
Just to add to this comment:
- EoS treats missing VRF and VRF `default` as the same. 

```
   ip access-group acl4_1 in
   ip access-group acl4_3 vrf default in
```
will end up in running-config as `ip access-group acl4_3 in` (last command overwrites previous ones)

- Order seems to be the following:
1. `tcp mss ceiling.*`
2. `ip access-group ingress default.*`
3. `ip access-group <IPv4_ACL_NAME> in` # IPv4 ACL for default` VRF. Can be only one
4. `ip access-group <IPv4_ACL_NAME> vrf <VRF_NAME> in` # Sorted by VRF name

Same ordering applies to IPv6.

Example:
```
avd-ci-leaf2(config-s-s19)#system control-plane
avd-ci-leaf2(config-s-s19-system-cp)#   tcp mss ceiling ipv4 1344 ipv6 1366
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group acl4_1 in
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group acl4_2 vrf red in
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group acl4_2 vrf red_1 in
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group acl4_3 vrf default in
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group ingress default acl4_4
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group acl4_5 vrf red_3 in
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group acl4_6 vrf red_5 in
avd-ci-leaf2(config-s-s19-system-cp)#   ip access-group ingress vrf red_4 in
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group acl6_1 in
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group acl6_2 vrf blue in
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group acl6_2 vrf blue_1 in
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group acl6_3 vrf default in
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group ingress default acl6_4
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group acl6_6 in
avd-ci-leaf2(config-s-s19-system-cp)#   ipv6 access-group ingress vrf blue_2 in
avd-ci-leaf2(config-s-s19-system-cp)#show session-config diffs 
--- system:/running-config
+++ session:/s19-session-config
+system control-plane
+   tcp mss ceiling ipv4 1344 ipv6 1366
+   ip access-group ingress default acl4_4
+   ip access-group acl4_3 in
+   ip access-group acl4_2 vrf red in
+   ip access-group acl4_2 vrf red_1 in
+   ip access-group acl4_5 vrf red_3 in
+   ip access-group ingress vrf red_4 in
+   ip access-group acl4_6 vrf red_5 in
+   ipv6 access-group ingress default acl6_4
+   ipv6 access-group acl6_6 in
+   ipv6 access-group acl6_2 vrf blue in
+   ipv6 access-group acl6_2 vrf blue_1 in
+   ipv6 access-group ingress vrf blue_2 in
avd-ci-leaf2(config-s-s19-system-cp)#
```